wiki/Community/Projects/en.md

```diff
@@ -83,25 +86,31 @@ Tools and/or services made by the osu! community. Feel free to add your own osu!
 ### Live streaming
 
 - gosumemory — Cross-platform memory reader for osu!
-  - [Github](https://github.com/l3lackShark/gosumemory)
+  - [GitHub](https://github.com/l3lackShark/gosumemory)
+- JKPS — A keys-per-second program that can be used as a input overlay with a lot of customization.
+  - [GitHub](https://github.com/JekiTheMonkey/JKPS)
 - Mikuia — A Twitch bot that can keep track of song requests and sends updates about any rank changes or plays you set in the chat.
   - [Website](https://mikuia.tv/)
-- [osu!Keys](https://osu.ppy.sh/community/forums/topics/497467) — Displays a key input overlay to overlay in a live stream.
-  - [GitHub](https://github.com/TheRedCoderGit/Osu-Keys/)
+- Ronnia — A alternative to Mikuia with a similar feature set
+  - [GitHub](https://github.com/aticie/ronnia) | [Website](https://ronnia.me/)
 - [osu!RequestBot](https://osu.ppy.sh/community/forums/topics/175723) — Keep track of song requests while streaming.
   - [GitHub](https://github.com/RedbackThomson/OsuRequestBot)
 - [osu!StreamCompanion](https://osu.ppy.sh/community/forums/topics/209616) — osu! information extractor for live streamers.
   - [GitHub](https://github.com/Piotrekol/StreamCompanion)
+- Ronnia — A alternative to Mikuia with a similar feature set
+  - [GitHub](https://github.com/aticie/ronnia) | [Website](https://ronnia.me/)
```

Comment:
this one's listed twice

Reply:
Forgot to remove it when I was sorting the newly added ones